### PR TITLE
[IMP] hr_skills: Rename hr_skill (OCA) to hr_skills (Odoo)

### PR DESCRIPTION
**Pull request by victoralmau**
Rename addon `hr_skills` (OCA) to `hr_skill` (Odoo)

Please @pedrobaeza review it

@Tecnativa TT26586

### CODE REVIEW

File: odoo/addons/openupgrade_records/lib/apriori.py [hunk 1]
```diff
@@ -7,7 +7,8 @@
     'crm_reveal': 'crm_iap_lead',
     'document': 'attachment_indexation',
     'payment_ogone': 'payment_ingenico',
-    # OCA/...
+    # OCA/hr
+    'hr_skill': 'hr_skills'
```
Comment (MiquelRForgeFlow) [inline odoo/addons/openupgrade_records/lib/apriori.py:12]:
```suggestion
    'hr_skills': 'hr_skill',
```
according to what your PR and commit message say

Comment (pedrobaeza) [reply odoo/addons/openupgrade_records/lib/apriori.py:12]: It's the contrary... so the title should be incorrect.

Comment (pedrobaeza) [inline odoo/addons/openupgrade_records/lib/apriori.py:12]: This is missing the commented TODO:
```suggestion
    # TODO: Transform possible data
    'hr_skill': 'hr_skills'
```

**Comment by MiquelRForgeFlow** [comment]
could you also edit commit message?

**Pull request merged**